common/src/main/java/org/valkyrienskies/mod/mixin/entity/MixinEntity.java

```diff
@@ -295,6 +296,17 @@ private void preDistanceToSqr(final double x, final double y, final double z,
         cir.setReturnValue(VSGameUtilsKt.squaredDistanceToInclShips(Entity.class.cast(this), x, y, z));
     }
 
+    /**
+     * @author ewoudje
+     * @reason use vs2 handler to handle this method
+     */
+    @Overwrite
```

Comment:
We can't use @Overwrite because it breaks Eureka dev environment because of a bug in Architectury. So we'll have to replace this with an @Inject.